src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -158,6 +158,26 @@ public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
             return this;
         }
 
+        /// <summary>
+        /// Specifies that the token refresh should only occur if the specified SHA-256 hash
```

Comment:
You need to provide clear guidance in the API docs on the encoding used to transform from byte[] to string, i.e. `Convert.ToBase64String`.